.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Apply to all files without commiting:
+#   pre-commit run --all-files
+# Update this file:
+#   pre-commit autoupdate
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      # ensure syntaxes are valid
+      - id: check-yaml
+      # catch git merge/rebase problems
+      - id: check-merge-conflict
+  - repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.28.5
+    hooks:
+      - id: check-github-workflows
+      # validate sub-teams schema
+      - id: check-jsonschema
+        files: ^teams/.*/.*\.ya?ml$
+        args: [--schemafile, teams/sub-teams-schema.json]
```

Comment:
Verifies whether the sub-team YAMLs are valid!

I recommend enabling [pre-commit CI](https://pre-commit.ci/)

Reply:
Xref #156